samples/snippets/get_operation.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Get a long-running operation.
+# [START contactcenterinsights_get_operation]
+from google.cloud import contact_center_insights_v1
+from google.longrunning import operations_pb2
+
+
+def get_operation(operation_name: str) -> operations_pb2.Operation:
```

Comment:
When a new user looks at this, will they know where to look to get the operation_name to use here?  Can you provide a comment that tells the user where to find this information?

Reply:
The user should know where to get the operation name because this snippet will be attached to [this public doc](https://cloud.google.com/contact-center/insights/docs/long-running-operations#get) that is a part of a how-to series on creating a topic model in Insights (recall that in order to create a long-running operation, the user will have to go through multiple steps that will eventually lead them to receiving an operation name from Insights). The [Google Long-running Operations API](https://google.aip.dev/151) is also a generic API that's widely used by Cloud services, and so I'd expect that most of our Cloud users are familiar with it. Let me know if that makes sense :)